umm-spec-lib/src/cmr/umm_spec/validation/spatial.clj

```diff
@@ -2,20 +2,30 @@
   "Defines validations for UMM spatial elements."
   (:require
    [camel-snake-kebab.core :as csk]
+   [clojure.string :as string]
    [cmr.common.util :as util]
+   [cmr.common.validations.core :as v]
    [cmr.spatial.validation :as sv]
-   [cmr.umm-spec.spatial-conversion :as sc]
-   [cmr.common.validations.core :as v]))
+   [cmr.umm-spec.spatial-conversion :as sc]))
 
 (def valid-coord-systems
-  "Coordinate systems that our valid for umm-spec geometries."
+  "Coordinate systems that are valid for umm-spec geometries."
   #{:geodetic :cartesian})
 
+(def valid-granule-spatial-representations
+  "Granule spatial representations that are valid for umm-spec spatial extents."
+  #{:geodetic :cartesian :orbit :no-spatial})
```

Comment:
There's a lot of converting to and from keywords, which I think makes the code harder to understand. I'd use the values we expect:
"GEODETIC" "CARTESIAN" "ORBIT" "NO_SPATIAL" throughout. Similarly for "GEODETIC" and "CARTESIAN" above. These enumerations should really be in the UMM-C (I assume we should be updating our latest UMM-C in addition to this change).

I think this should be a UMM-C change to add the enumerations and then add a migration to set these values correctly for old versions of UMM-C.

Reply:
done